src/main/java/sage/Command/AddCommand.java

```diff
@@ -0,0 +1,97 @@
+package sage.Command;
+
+import sage.List.TaskList;
+import sage.SageException;
+import sage.Storage;
+import sage.Task.DeadlineTask;
+import sage.Task.EventTask;
+import sage.Task.Task;
+import sage.Task.ToDoTask;
+import sage.Ui;
+
+public class AddCommand extends Command {
```

Comment:
I like how clean and organised your code is! The different classes for each type of command makes it very clear 🔥